Pod/Sources/Categories/UIViewController+OriginateTabBarScroller.m

```diff
@@ -21,6 +22,17 @@ - (CGFloat)ori_previousScrollViewOffset
     return [objc_getAssociatedObject(self, @selector(ori_previousScrollViewOffset)) floatValue];
 }
 
+// The initial content offset
+- (void)setOri_InitialContentOffset:(CGFloat)contentOffset
```

Comment:
Doesn't really matter since it's all internal, but technically should be `-setOri_initialContentOffset:`